src/test/resources-noncompilable/com/puppycrawl/tools/checkstyle/checks/outertypefilename/InputOuterTypeFilenameRecord.java

```diff
@@ -1,8 +1,8 @@
 //non-compiled with javac: contains different class name by demand of test
 package com.puppycrawl.tools.checkstyle.checks.outertypefilename;
+
 /* Config:
  * default
  */
-public record IncorrectName(int x, int y, String str) { // violation
-
+public record IncorrectName1(int x, int y, String str) { // violation
```

Comment:
plase move this update  in separate first "minor: ......" commit.
To explain why it is required.

Reply:
Done.